pom.xml

```diff
@@ -75,7 +75,7 @@
     <version.io.opentracing.contrib-opentracing-spring-tracer-configuration-starter>0.1.0</version.io.opentracing.contrib-opentracing-spring-tracer-configuration-starter>
     <version.io.opentracing.contrib-opentracing-spring-web>1.1.1</version.io.opentracing.contrib-opentracing-spring-web>
     <version.io.opentracing.contrib-opentracing-spring-jms>0.0.10</version.io.opentracing.contrib-opentracing-spring-jms>
-    <version.io.opentracing.contrib-opentracing-spring-messaging>0.0.5</version.io.opentracing.contrib-opentracing-spring-messaging>
+    <version.io.opentracing.contrib-opentracing-spring-messaging>0.1.2</version.io.opentracing.contrib-opentracing-spring-messaging>
```

Comment:
Is it using OT API 0.32.0?

Reply:
Yes

Reply:
Could you please then open PR to #212 so we keep the same versions in master?

Reply:
@pavolloffay https://github.com/pavolloffay/java-spring-cloud/pull/2

Reply:
@pavolloffay I've raised a new PR (https://github.com/pavolloffay/java-spring-cloud/pull/3) from a different branch, to not confuse things too much in here.